cfg/constants.go

```diff
@@ -24,3 +24,14 @@ const (
 	ERROR   string = "ERROR"
 	OFF     string = "OFF"
 )
+
+const (
+	// File Cache Config constants.
+
+	DefaultFileCacheMaxSizeMB       = -1
+	DefaultEnableCRC                = false
+	DefaultEnableParallelDownloads  = false
+	DefaultDownloadChunkSizeMB      = 50
+	DefaultParallelDownloadsPerFile = 16
+	DefaultCacheFileForRangeRead    = false
```

Comment:
Please review if these constants are being used anywhere.

Reply:
This was only being used in the old config package after removing the constants from tests. Moved it to old config package. Thanks for catching this!